streamer/src/sendmmsg.rs

```diff
@@ -73,40 +73,32 @@ fn mmsghdr_for_packet(
     const SOCKADDR_IN6_PADDING: usize = SIZE_OF_SOCKADDR_STORAGE - SIZE_OF_SOCKADDR_IN6;
 
     iov.write(iovec {
-        iov_base: packet.as_ptr() as *mut libc::c_void,
+        iov_base: packet.as_ptr().cast_mut().cast(),
```

Comment:
nit: I think we may be able to do this to save one cast:
```rust
packet.as_mut_ptr().cast()
```

https://doc.rust-lang.org/src/core/ptr/mut_ptr.rs.html#44-46

Reply:
Oops, you're right